metadata.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+- major: 0
+  minor: 5
+  contract: v1alpha4
```

Comment:
if this is temporarily set to v1a3, we could test and merge this PR before the CAPI uplift, and change it during capi uplift

Reply:
Good point, I ll do the change

Reply:
Changed